src/utils/strings.ts

```diff
@@ -44,3 +44,17 @@ export const textShortener = ({ charsEnd = 10, charsStart = 10, ellipsis = '...'
  * @returns {string} string without side whitespaces
  */
 export const trimSpaces = (value: string): string => (value === undefined ? '' : value.trim())
+
+/**
+ * Util to compare two strings, comparison is case insensitive
+ * @param str1
+ * @param str2
+ * @returns {boolean}
+ */
+export const sameString = (str1: string | undefined, str2: string | undefined): boolean => {
+  if (!str1 || !str2) {
```

Comment:
what if both are empty string? :grimacing: or it's a case intentionally left out?

Reply:
`sameAddress` has the same approach. Well, maybe it's better that way to not break anything else

Reply:
yes.. they are the same string literal, it would be incorrect to return false IMHO.